src/geotiffimage.js

```diff
@@ -92,49 +92,55 @@ class GeoTIFFImage {
   getFileDirectory() {
     return this.fileDirectory;
   }
+
   /**
    * Returns the associated parsed geo keys.
    * @returns {Object} the parsed geo keys
    */
   getGeoKeys() {
     return this.geoKeys;
   }
+
   /**
    * Returns the width of the image.
    * @returns {Number} the width of the image
    */
   getWidth() {
     return this.fileDirectory.ImageWidth;
   }
+
   /**
    * Returns the height of the image.
    * @returns {Number} the height of the image
    */
   getHeight() {
     return this.fileDirectory.ImageLength;
   }
+
   /**
    * Returns the number of samples per pixel.
    * @returns {Number} the number of samples per pixel
    */
   getSamplesPerPixel() {
     return this.fileDirectory.SamplesPerPixel;
   }
+
   /**
    * Returns the width of each tile.
    * @returns {Number} the width of each tile
    */
   getTileWidth() {
     return this.isTiled ? this.fileDirectory.TileWidth : this.getWidth();
   }
+
   /**
    * Returns the height of each tile.
    * @returns {Number} the height of each tile
    */
   getTileHeight() {
     if (this.isTiled) {
       return this.fileDirectory.TileLength;
-    } else if (typeof this.fileDirectory.RowsPerStrip !== 'undefined') {
```

Comment:
I didn't change this line manually this is related to `npm run lint:fix`. We should modify eslint rules if you prefer the other syntax otherwise `npm run lint` will probably fail.

Reply:
I just came back to that line in the source code, I agree that hurts 😆
I'll add a newline at least that will not change anything to eslint validation, let me know if you want me to change it back to `else if`.

Reply:
I've applied the change to every occurence of that ugly `if ... } if`